src/lighthouse.php

```diff
@@ -126,6 +126,37 @@
         'tags' => false,
     ],
 
+    /*
+    |--------------------------------------------------------------------------
+    | Query Cache
+    |--------------------------------------------------------------------------
+    |
+    | Caches user queries so we won't have to parse string queries on each request
+    |
+    */
+
+    'query_cache' => [
+        /*
+         * Setting to true enables query caching.
+         */
+        'enable' => env('LIGHTHOUSE_QUERY_CACHE_ENABLE', 'local' !== env('APP_ENV')),
```

Comment:
why not local?

Reply:
I just copied it from schema caching. It works on any env and all tests are passed with enabled query cache.

Reply:
```suggestion
        'enable' => env('LIGHTHOUSE_QUERY_CACHE_ENABLE', true),
```